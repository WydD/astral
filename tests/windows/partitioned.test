<?xml version="1.0" encoding="UTF-8"?>
<astral xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://astral.ligforge.imag.fr/schema/query.xsd">
<query t0="auto">
<window partitionedBy="id">
<description wtype="L"/>
<source attributes="id,T,PHYID,temp" id="S" sourceAttributes="id,T,PHYID,temp" type="Stream"/>
</window>
</query>
<sources>
<source attributes="id,T,PHYID,temp" file="tests/windows/S.S" id="S" impl="FileStreamImpl" type="Stream"/>
</sources>
</astral>
