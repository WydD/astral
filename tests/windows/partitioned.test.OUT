id,T,temp
t=1000
1,1000,30
t=1005
1,1005,35
t=1006
1,1005,35
2,1006,32
t=1007
2,1006,32
1,1007,25
t=1009
1,1009,29
2,1009,30
t=1010
2,1009,30
1,1010,33
t=1012
2,1009,30
1,1012,25
t=1013
2,1009,30
1,1013,25
t=1015
2,1009,30
1,1015,32
t=1020
2,1020,24
1,1020,35
t=1021
2,1020,24
1,1021,27
t=1025
2,1020,24
1,1025,27
t=1026
2,1020,24
1,1026,35
t=1035
1,1026,35
2,1035,27
